pkg/controllers/dynakube/extension/databases/deployment.go

```diff
@@ -37,6 +37,20 @@ const (
 	certsMountPath     = "/var/ssl-certs/dynatrace"
 )
 
+// ListDeployments returns a list of database datasource deployments that are managed by the DynaKube.
+func ListDeployments(ctx context.Context, clt client.Reader, dk *dynakube.DynaKube) ([]appsv1.Deployment, error) {
+	deployments := &appsv1.DeploymentList{}
+
+	// We have to build labels from an empty DB spec to allow cleaning up orphans (without having to delete the DynaKube).
+	deploymentLabels, _, _ := buildAllLabels(dk, extensions.DatabaseSpec{})
+
+	if err := clt.List(ctx, deployments, client.InNamespace(dk.Namespace), sanitizedListLabels(deploymentLabels)); err != nil {
+		return nil, fmt.Errorf("list deployments: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to list deployments: %w", err)
```

Reply:
I generally wrap errors according to the [uber style guide](https://github.com/uber-go/guide/blob/master/style.md#error-wrapping). Less noise in the error messages